assets/scss/partials/layout/center-col.scss

```diff
@@ -1,15 +1,10 @@
 .column-center {
-
-  --col-width: 60%;
-
-  width: var(--col-width);
-  float: left;
   border-bottom: 1px solid rgba(30,44,94,.2);
 
   &__full-width-container{
     --padding: 5%;
-    margin-left: calc( (var(--col-width)/2 * -1) + var(--padding) );
-    width: calc(100% + var(--col-width) - (2 * var(--padding)));
+    margin-left: calc( (var(60%)/2 * -1) + var(--padding) );
+    width: calc(100% + var(60%) - (2 * var(--padding)));
```

Comment:
`var()` in CSS can only contain CSS variables, like `--col-width`, not actual property values, like `60%`. If you inspect the divs that have the `__full-width-container` class in your browser, you'll see that both `margin-left` and `width` are now invalid properties.